lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -224,16 +228,27 @@ class Simulator {
 
     if (node.type !== Node.TYPES.NETWORK) throw new Error('Unsupported');
 
-    const record = /** @type {NetworkNode} */ (node).record;
+    const networkNode = /** @type {NetworkNode} */ (node);
     const timingData = this._nodeTimings.get(node);
-    // If we're estimating time remaining, we already acquired a connection for this record, definitely non-null
-    const connection = /** @type {TcpConnection} */ (this._acquireConnection(record));
-    const calculation = connection.simulateDownloadUntil(
-      record.transferSize - timingData.bytesDownloaded,
-      {timeAlreadyElapsed: timingData.timeElapsed, maximumTimeToElapse: Infinity}
-    );
 
-    const estimatedTimeElapsed = calculation.timeElapsed + timingData.timeElapsedOvershoot;
+    let timeElapsed = 0;
+    if (networkNode.fromDiskCache) {
+      // Rough access time for seeking to location on disk and reading sequentially
+      // @see http://norvig.com/21-days.html#answers
+      const sizeInMb = (networkNode.record._resourceSize || 0) / 1024 / 1024;
+      timeElapsed = 8 + 20 * sizeInMb - timingData.timeElapsed;
```

Comment:
I think this is slower in Chrome. UMA must have some timings we could look at.

Reply:
yeah we're not really accounting for the full Chrome IPC RT time, just the OS/disk access, but the same could be said for network really

perhaps we add an IPC RT latency factor in the future where we account for more based on the chunk size of all IPC-based traffic?

Reply:
okay yah sg. `DiskCache.0.AsyncIOTime` is the best one but it doesn't give us much as its measured from the cache thread and no cost to browser IO thread or over the IPC to the renderer.